'make test' fails out-of-the-box

After a fresh clone of the project and after running `make compose`, I can't get unit tests to work. Since this is my first go at building an app, I'm completely lost for what to do next.

```
./vendor/phpunit/phpunit/phpunit -c phpunit.xml
PHP Warning:  require_once(/var/www/nextcloud/apps/app-tutorial/tests/../../../tests/bootstrap.php): Failed to open stream: No such file or directory in /var/www/nextcloud/apps/app-tutorial/tests/bootstrap.php on line 3
PHP Fatal error:  Uncaught Error: Failed opening required '/var/www/nextcloud/apps/app-tutorial/tests/../../../tests/bootstrap.php' (include_path='.:/usr/share/php') in /var/www/nextcloud/apps/app-tutorial/tests/bootstrap.php:3
Stack trace:
#0 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/Util/FileLoader.php(66): include_once()
#1 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/Util/FileLoader.php(54): PHPUnit\Util\FileLoader::load()
#2 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/TextUI/Command.php(1126): PHPUnit\Util\FileLoader::checkAndLoad()
#3 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/TextUI/Command.php(922): PHPUnit\TextUI\Command->handleBootstrap()
#4 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/TextUI/Command.php(202): PHPUnit\TextUI\Command->handleArguments()
#5 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/src/TextUI/Command.php(194): PHPUnit\TextUI\Command->run()
#6 /var/www/nextcloud/apps/app-tutorial/vendor/phpunit/phpunit/phpunit(98): PHPUnit\TextUI\Command::main()
#7 {main}
  thrown in /var/www/nextcloud/apps/app-tutorial/tests/bootstrap.php on line 3
make: *** [Makefile:70: test] Error 255

```

--- a/README.md
+++ b/README.md
@@ -11,13 +11,15 @@ To install it change into your Nextcloud's apps directory:
 
     cd nextcloud/apps
 
-Then run:
+Then clone this repository into a folder named **notestutorial**¹:
 
     git clone https://github.com/nextcloud/app-tutorial.git notestutorial
 
 Then install the dependencies using:
 
     make composer
+
+¹ It is important that the directory is named exactly like the app ID (see `appinfo/info.xml`).
 
 ## Frontend development
 

